go.mod

```diff
@@ -5,13 +5,7 @@ go 1.17
 // Legacy tags before v0.1.0 was created
 retract [v1.4.0-pre-alpha-1, v1.4.0-pre-alpha-2]
 
-replace (
-	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5 // indirect
-	github.com/k14s/kbld => github.com/anujc25/carvel-kbld v0.31.0-update-vendir
-	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.1
-	sigs.k8s.io/cluster-api/test => sigs.k8s.io/cluster-api/test v1.0.1
-	sigs.k8s.io/kind => sigs.k8s.io/kind v0.11.1
-)
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.1
```

Comment:
Do we still need this `replace` if `cluster-api` version in L102 is the same? Or is this due to `cluster-api/test` needing it?